pkg/backup/compaction_processor.go

```diff
@@ -261,6 +266,26 @@ func (p *compactBackupsProcessor) processSpanEntries(
 			logClose(ctx, sink, "SST sink")
 		}
 	}()
+	// Passing a nil pacer is effectively a noop if CPU control is disabled.
+	var pacer *admission.Pacer
```

Comment:
this looks identical to the backup processor code. could we instead call a common helper: `newBackupPacer(clusterSettings)`?

Reply:
Do you think there should be any unified place for us to place all these helpers? So far I've just been leaving the helpers wherever I define them and since it all goes in one package then it's fine, but they are sort of scattered around. My knee-jerk reaction is to create a `backup_utils.go` file, but I find that `utils` files tend to be a bit of a code smell as it basically becomes a "miscellaneous" drawer for us to throw everything into.